requirements/requirements-base.txt

```diff
@@ -2,3 +2,4 @@ Django>=1.7.5
 python-etcd>=0.4.1
```

Comment:
Do you need any of the other requirements still? I think you have them all in the etcd-config already and you don't explicitly use them anymore correct?